### PR TITLE
cleanup tags and when statements

### PR DESCRIPTION
**Pull request by avgils**

### CODE REVIEW

File: roles/django/tasks/main.yml [hunk 1]
```diff
@@ -60,32 +50,22 @@
...
   template: src=../templates/02_django.j2 dest=/etc/httpd/conf.d/02_django.conf owner=apache group=apache mode=0644
-  tags:
-    - install
 
 - name: Install custom error pages into apache
   become: yes
   template: src=../templates/03_custom-error-pages.j2 dest=/etc/httpd/conf.d/03_custom-error-pages.conf owner=apache group=apache mode=0644
-  tags:
-    - install
+
```
Comment (evetion) [inline roles/django/tasks/main.yml:71]: Weird no newline?

File: roles/postgresql/tasks/main.yml [hunk 2]
```diff
@@ -43,49 +41,37 @@
...
                     password={{ djangodbpass }}
-  tags:
-    - user
 
 - name: change django dev user
   become: yes
   become_user: postgres
   postgresql_user:  db=djangodb
                     name=django
                     role_attr_flags="CREATEDB"
```
Comment (evetion) [inline roles/postgresql/tasks/main.yml:74]: Required for running tests

File: site_k8s.yml [hunk 3]
```diff
@@ -9,10 +9,10 @@
...
     - { role: k8s_addons }
 
-- name: Install delft3D-GT Django 
-  hosts: delft3dgt-dev, delft3dgt-accp, delft3dgt-test, delft3dgt-prod
-  roles:
-    - { role: django_app, tags: 'app' }
-    - { role: k8s_connection}
-    - { role: cloud_amazon}
-    - { role: thredds, tags: 'thredds' }
+# - name: Install delft3D-GT Django
```
Comment (evetion) [inline site_k8s.yml]: Why all the comments?

File: .travis.yml [hunk 4]
```diff
@@ -5,8 +5,8 @@ install:
 - pip install -r requirements.txt
 - ansible-galaxy install Datadog.datadog
 script:
-- ansible-playbook site.yml --check
-- ansible-playbook site_amazon.yml --check
+- ansible-playbook site_k8s.yml --syntax-check
```
Comment (evetion) [inline .travis.yml:8]: Why not use --check?

Comment (avgils) [reply .travis.yml:8]: Because check is failing on all amazon stuff. It doesn't really run so nothing is registered.

File: site_k8s.yml (hunk 3 above)
Comment (avgils) [reply site_k8s.yml]: oops, thanks

**Pull request merged**